src/config.js

```diff
@@ -75,6 +75,9 @@ export const urls = {
   localhostRegex: /^https?:\/\/localhost(:\d+)?$/,
   errorReporting:
     env['errorReportingUrl'] || 'https://amp-error-reporting.appspot.com/r',
+  betaErrorReporting:
```

Comment:
This needs to be added to externs: build-system/externs/amp.extern.js

Reply:
Done.